x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -10,18 +10,34 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const pageObjects = getPageObjects(['common', 'apiKeys']);
   const log = getService('log');
+  const security = getService('security');
+  const find = getService('find');
 
   describe('Home page', function() {
     this.tags('smoke');
     before(async () => {
+      await security.testUser.setRoles(['kibana_admin']);
       await pageObjects.common.navigateToApp('apiKeys');
     });
 
+    after(async () => {
+      await security.testUser.restoreDefaults();
+    });
+
+    // https://www.elastic.co/guide/en/kibana/7.6/api-keys.html#api-keys-security-privileges
+    it('Shows required privileges ', async () => {
+      log.debug('Checking for section header');
+      const messageElement = await find.byCssSelector('h2.euiTitle');
+      const message = await messageElement.getVisibleText();
+      expect(message).to.be('You need permission to manage API keys');
```

Comment:
Maybe the 2 lines;
find.byCssSelector('h2.euiTitle');
messageElement.getVisibleText();

should go in a method in pageObjects.apiKeys

Would also be nice if we could add a data-test-subj for that text.

Reply:
Or maybe it's a shared thing across many apps and should be a common service or in header_page ?

Reply:
I added a `data-test-subj` specific to the permission denied page, as it is specific to that page and the message is differing from app to app.